tests/testthat/test-log-import.R

```diff
@@ -39,8 +39,8 @@ test_that("postgres log data is persisted", {
   expect_equal(postgres_data$name, "example")
   expect_equal(postgres_data$language, "r")
   expect_equal(postgres_data$mode, "append")
-  expect_equal(postgres_data$start_time, as.POSIXct(NA))
-  expect_equal(postgres_data$end_time, as.POSIXct(NA))
+  expect_equal(postgres_data$start_time, as.POSIXct(NA, tz = "UTC"))
+  expect_equal(postgres_data$end_time, as.POSIXct(NA, tz = "UTC"))
```

Comment:
This has started failing, assuming some change in DBI when reading time data or in expect_equal.